index.html

```diff
@@ -0,0 +1,128 @@
+                        <img id="TwitterIcon" src="images/Icons/Twitter.png" alt="Twitter Icon">
+                    </a>
+                    <a id="englishLink" href="javascript:void(0);" class="" >English</a>
+                    <a id="myPagaLink" href="javascript:void(0);" class="" >My Page</a>
+                    <a id="logoutLink" href="javascript:void(0);" class="" >Logout</a>
+                </div>
+                <!--Menu Text-->
+                <div class="menuTextContainer generalPadding">
+                    <div id="myLogo" class="myLogo">
+                        <a id="homeLogo" class="logoContainer">
+                            <img src="images/Icons/Logo.png" alt="World Tour Logo">
+                        </a>
+                    </div>
+                    <div id="myLinks" class="textLinks menuText myFont">
+                        <a id="closeHamburger" href="javascript:void(0);" class="icon left">
+                            <img id="closeMenuImage" src="images/xMenu_50.png" alt="Close Menu">
+                        </a>
+                        <a id="home" href="#home" tabindex="1">Home</a>
+                        <a id="about" href="#about" tabindex="1">About</a>
+                        <a href="#Program" tabindex="2">Program</a>
+                        <a href="#Join" tabindex="3">Join</a>
+                        <a href="#Sponsor" tabindex="4">Sponsor</a>
+                        <a href="#News" tabindex="5">News</a>
+                    </div>
+                    <div class="importantLink">
+                        <a href="#contact" tabindex="5">CC Campaign</a>
+                    </div>
+                </div>
+            </div>
+        </header>
+        <section>
+            <div id="main" class="generalPadding main">
+                <!--Background image-->
+                <div  id="mainBackground" class="mainBackground"></div>
+                <!--Menu Hamburger-->
+                <div id="menuHamburgerContainer" class="menuHamburgerContainer">
+                    <a id="menuHamburger" href="javascript:void(0);" class="menuHamburger">
+                        <img id="baraMenuImage" src="images/barMenu.png" alt="Hamburger Menu" class="">
+                    </a>
+                </div>
+                <div class="mainContainer myFont">
+                    <h2>"Hello! Sharing world"</h2>
+                    <h1>CREATIVE COMMONS<br>GLOBAL SUMMIT<br>2015</h1>
+                    <p class="darkerColor">A joyful celebration believing in the value of openess and sharing, creating a positive change with people from over 80 countries is taking place in October, in Korea</p>
+                    <h3 class="darkerColor">2015.10.15&#40;THU&#41; ~ 16&#40;FRI&#41;</h3>
+                    <H4>&#64; National Museum Korea, Art Center Nabi and more</H4>
+                </div>
+            </div>
+        </section>
+        <section class="kinds generalPadding myFont">
```

Comment:
- [ ] [REQUIRED] Good job so far! Please give `hover effect` to each card like a [Behance Design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏🏼 

![image](https://github.com/alexansaa/ConcertAdv/assets/63915024/c4775396-1686-44bc-9d14-6e618e99d369)